src/unstructured_client/_hooks/custom/log_retries_after_error_hook.py

```diff
@@ -11,29 +11,25 @@
     SDKInitHook,
 )
 
+logger = logging.getLogger(UNSTRUCTURED_CLIENT_LOGGER_NAME)
+
 
 class LogRetriesAfterErrorHook(AfterErrorHook, SDKInitHook):
     """Hook providing visibility to users when the client retries requests"""
 
     def log_retries(self, response: Optional[requests.Response]):
         """Log retries to give users visibility into requests."""
-        logger = logging.getLogger(UNSTRUCTURED_CLIENT_LOGGER_NAME)
-        logger.setLevel(logging.INFO)
-
-        if response is not None and response.status_code == 500:
-            logger.info("Response status code: 500. Sleeping before retry.")
 
-            if bool(response.text):
-                logger.info(response.text)
+        if response is not None and response.status_code // 100 == 5:
+            logger.info("Failed to process a request due to API server error with status code %d."
+                        "Sleeping before retry.",
+                        response.status_code)
+            if response.text:
+                logger.info("Server message - %s", response.text)
```

Comment:
`openapi.json` defines that retry should happen on 5XX codes rather then only 500, I've modified logs to match that but I don't know which intention is the desired one.
```json
    "x-speakeasy-retries": {
        "strategy": "backoff",
        "backoff": {
            "initialInterval": 500,
            "maxInterval": 60000,
            "maxElapsedTime": 900000,
            "exponent": 1.5
        },
        "statusCodes": [
            "5xx"
        ],
        "retryConnectionErrors": true
    }
```